src/quo2/components/settings/settings_list/view.cljs

```diff
@@ -0,0 +1,133 @@
+(ns quo2.components.settings.settings-list.view
+  (:require [quo2.components.settings.settings-list.style :as style]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.selectors.selectors.view :as selectors]
+            [quo2.components.buttons.button :as button]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]))
+
+(defn settings-title
+  [title]
+  [rn/view
+   {:flex 1}
```

Comment:
Always add style values inside the `:style` key.
https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#always-add-styles-inside-the-style-key

Reply:
Absolutely agree, sorry.